### ISSUE TITLE
Table not created during migration

### ISSUE THREAD

**Issue by designermonkey** [cakephp/phinx#1295]
If I use the following:

```php
        $table->addColumn('modified_at', 'timestamp', [
            'null' => false,
            'default' => 'CURRENT_TIMESTAMP',
            'update'  => 'CURRENT_TIMESTAMP',
            'precision' => 4,
        ]);
```

The use of `update` causes the migration of the entire table to fail.

Tested with SQLite as the database engine.

**Comment by MasterOdin** [comment]
Is there anything special about where you define `$table` or is it just something like `$table = $this->table('test_table');`?

**Comment by designermonkey** [comment]
That’s exactly it yes. I followed the examples in the documentation.

**Comment by dereuromark** [comment]
You seem to have forgotten the final $table->update() or $table->create() statement.
I dont like that they are needed - they should be auto-used at the end based on the existence of them.
But as for now, closing as user error.

**Comment by designermonkey** [comment]
Erm, it's not a user error!

My example is a very simplified example of the specific column that causes the entire migration to fail, as I alluded to in the initial report!

As I have stated, I have already nailed down the issue for you: the use of `update` in a column definition. It has nothing to do with forgetting `update()` or `create()`.

**Comment by dereuromark** [comment]
How can this be reproduced? We all use it this way and we do not see this issue.
Please provide a way to reproduce it.

**Comment by shahariaazam** [comment]
Yes, `CURRENT_TIMESTAMP` as default value won't work in SQLite. Two way we can solve that.
- On create phinx can ignore `CURRENT_TIMESTAMP` as column default value if the adapter type is `sqlite`
- Or we can simply implement login in our migration file..
```php
->addColumn('added_on', 'datetime', ['default' => $this->getAdapter()->getAdapterType() === "mysql" ? "CURRENT_TIMESTAMP" : null])
```
